RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate DefaultBrowserView to Typescript

## Proposed changes


## Issue(s)	


## How to test or reproduce

- Go to `Settings`
- You will see a `Default browser` button, then click on it and you will move to `DefaultBrowserView`
- There, the layout should be rendered properly, besides the function must work too

## Screenshots

## Types of changes




## Checklist



## Further comments
